portfolio/src/main/webapp/script.js

```diff
@@ -43,19 +55,45 @@ async function getUserComments(){
     }
 }
 
+/* Delete all the comments from the server */
+async function deleteComments(){
+    try{
+        await fetch("/delete-data", {
+            method: "POST"
+        });
+    }
+    catch(error){
```

Comment:
nit: have the catch on the same line as the closing bracket and add spaces 
try {
 ...
} catch(...) {
} finally {
}